app/styles/HeaderComp.scss

```diff
@@ -0,0 +1,97 @@
+* {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  text-decoration: none;
+}
+.header {
+  width: 100%;
+  display: block;
+  padding: 5px;
+  box-shadow: 0px 0px 5px gray;
+  position: relative;
+  z-index: 9;
+}
+.inner-header {
+  width: 100%;
+  max-width: 1200px;
+  display: block;
+  margin: 0 auto;
+  // border: 1px solid red;
+}
+.logo-container {
+  width: 190px;
+  float: left;
+  // display: table;
+}
+.navigation {
+  margin-left: 80px;
+  float: left;
+  font-weight: $FontWeightMedium;
+  // padding-top: 50px;
+}
+.navigation li {
```

Comment:
you are using scss, li of navigation could be wrapped inside above navigation rule itself